### PR TITLE
feat: create subcourse group chat

### PR DESCRIPTION
**Pull request by LomyW**

### CODE REVIEW

File: src/pages/student/SingleCourseStudent.tsx [hunk 1]
```diff
@@ -121,7 +121,7 @@ const SingleCourseStudent = () => {
     const { t } = useTranslation();
     const { space, sizes } = useTheme();
     const toast = useToast();
-
+    const navigate = useNavigate();
```
Comment (Jonasdoubleyou) [inline src/pages/student/SingleCourseStudent.tsx:124]: (Why is this added?)

Comment (Jonasdoubleyou) [inline src/pages/student/SingleCourseStudent.tsx:124]: (Why is this added?)

**Review by Jonasdoubleyou — APPROVED** [review]

File: src/pages/subcourse/OpenSubcourseChat.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    const [createSubcourseGroupChat] = useMutation(
+        gql(`
+            mutation createGroupChat($subcourseId: Float!) {
+                subcourseGroupChatCreate(subcourseId: $subcourseId)
+            }
+        `)
+    );
+
+    const openSubcourseGroupChat = async () => {
+        const conversation = await createSubcourseGroupChat({ variables: { subcourseId: subcourseId ?? 1 } });
```
[the remark this answers is not in the document]
Comment (LomyW) [reply src/pages/subcourse/OpenSubcourseChat.tsx:26]: we can't create the group chat on subcourse creation, because we need more than to members to create the group chat. So we have to wait until two pupils joined the subcourse.

[the remark this answers is not in the document]
Comment (LomyW) [reply src/pages/subcourse/OpenSubcourseChat.tsx:26]: so check if conversationId is not null? And if it is null than use the mutation?

[the remark this answers is not in the document]
Comment (Jonasdoubleyou) [reply src/pages/subcourse/OpenSubcourseChat.tsx:26]: Yes exactly

**Review by Jonasdoubleyou — APPROVED** [review]
nice!
